src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/OrderlyServerAPI.kt

```diff
@@ -38,12 +39,18 @@ interface OrderlyServerAPI
     fun get(url: String, context: ActionContext): OrderlyServerResponse
 
     @Throws(OrderlyServerError::class)
-    fun get(url: String, queryParams: Map<String, String>): OrderlyServerResponse
+    fun get(url: String, queryParams: Map<String, String?>): OrderlyServerResponse
```

Comment:
Changed the type of this parameter, so could pass `context.queryParams()` directly in from the ReportController. I don't think it should have any ill effects.

Reply:
Which implementations? The clients which want to send `Map<String, String>` only, i.e. the ones that build the query params themselves, can still do so. This just allows the general case and fixes an inconsistency where `queryParams()`in the context is `Map<String, String?>` - and that can be passed straight through if you use the overload with the context parameter - but you couldn't pass `context.queryParams()` itself here.

Reply:
Actually looking at the `queryParams()` function in `DirectActionContext`, I don't think it could ever return null values. The nullable type is an artefact of the fact that in general you can't assume a value exists for arbitrary keys in a map, but as the keys of the map are generated from `servletRequest.getParameterMap().keySet()` we *can* be confident that values for them exist. Could we just change the signature of the function in `ActionContext` to be `Map<String, String>`?

Reply:
That makes sense, it would presumably give an empty string for empty params.

Reply:
I think so. Maybe worth checking. The underling `servletRequest` function to get a query parameter value says it returns null if a key does not exist.